### ISSUE TITLE
Is this project dead? Any guide or docs to overtake this project?

### ISSUE THREAD

**Issue by NerijusD** [Elfocrash/L2dotNET#76]

**Comment by Elfocrash** [comment]
Yeah it's inactive and there isn't really and documentation around it. It was mostly an experiment. You can fork it and try to spin it up but the latest rev is in a really flaky state.
